css/style.css

```diff
@@ -0,0 +1,855 @@
+body {
+  padding: 0;
+  margin: 0; }
+
+main {
+  margin: 0;
+  padding: 0; }
+
+ul,
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0; }
+
+input,
+button {
+  outline: none; }
+
+button,
+a {
+  cursor: pointer;
+  text-decoration: none;
+  font-family: "Rajdhani", sans-serif; }
+
+a {
+  font-family: "Rajdhani", sans-serif; }
+
+.flex {
+  display: flex; }
+
+.flex-column {
+  display: flex;
+  flex-direction: column; }
+
+.j-c-center {
+  justify-content: center;
+  align-items: center; }
+
+.j-c-end {
+  justify-content: end; }
+
+.j-c-space-between {
+  justify-content: space-between; }
+
+.hide {
+  display: none; }
+
+.fixed {
+  position: fixed;
+  z-index: 55;
+  width: 100%;
+  margin-right: 20px; }
+
+.open_menu {
+  display: block;
+  transition: all 0.5s ease-in-out;
+  animation: slide_in_from_rigth 0.5s forwards; }
+
+.close_menu {
+  transition: all 0.5s ease-in-out;
+  animation: slide_out_to_right 0.5s forwards; }
+
+.container {
+  width: 100%;
+  padding: 0 20px;
+  margin: 0 auto;
+  box-sizing: border-box; }
+  @media (max-width: 420px) {
+    .container {
+      width: 100%; } }
+  @media (max-width: 576px) {
+    .container {
+      width: 100%;
+      padding: 0 !important; } }
+  @media (max-width: 812px) {
```

Comment:
Do not use the `!important`, it is not a good practice, this word should only be used in very special cases.